akka-docs/src/main/paradox/scala/persistence.md

```diff
@@ -1186,8 +1186,10 @@ Scala
     "org.fusesource.leveldbjni"   % "leveldbjni-all"   % "1.8"
     ```
     @@@
-    @@@vars
-     ```
+
+Java
```

Comment:
Maybe sbt/maven would be better as headers instead of scala/java, but this is a nice improvement anyway - thanks!